nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Discover/DiscoveryWorkerTests.cs

```diff
@@ -709,6 +709,127 @@ await TestDiscoveryAsync(
         );
     }
 
+    [Fact]
+    public async Task TestRepo_DirectDiscovery_Slnx()
+    {
+        var solutionPath = "solution.slnx";
+        await TestDiscoveryAsync(
+            experimentsManager: new ExperimentsManager() { UseDirectDiscovery = true },
+            packages:
+            [
+                MockNuGetPackage.CreateSimplePackage("Some.Package", "9.0.1", "net7.0"),
+            ],
+            workspacePath: "",
+            files: new[]
+            {
+                ("src/project.csproj", """
+                    <Project Sdk="Microsoft.NET.Sdk">
+                      <PropertyGroup>
+                        <TargetFrameworks>net7.0;net8.0</TargetFrameworks>
```

Comment:
Just a single target framework should suffice since the desired behavior is verifying `.slnx` expansion and nothing else.